src/main/java/me/karakelley/tictactoe/GameConfiguration.java

```diff
@@ -0,0 +1,48 @@
+package me.karakelley.tictactoe;
+
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+
+public class GameConfiguration {
+  private String player1marker;
+  private String player2marker;
+  private String path = System.getProperty("configuration");
+  private Parser parser;
+
+  public GameConfiguration() {
+    this.parser = setupParser();
+    this.player1marker = "X";
+    this.player2marker = "O";
+  }
+
+  public String getPlayer1marker() {
+    setPlayerMarkers();
+    return player1marker;
+  }
+
+  public String getPlayer2marker() {
+    setPlayerMarkers();
+    return player2marker;
+  }
+
+  private Parser setupParser() {
+    if (path != null) {
+      try {
+        parser = new Parser(new FileReader(path), new JSONParser());
```

Comment:
how come you're not injecting your Parser? What if you want to use another parser, that capitalizes all letters or something?

Reply:
I originally injected it but found if the path was null, meaning there is no config file, then I don't actually need a Parser and I just want the defaults for player markers. If I injected it and the path was null then I would get errors. I had a hard time designing this story.